editor/plugins/editor_resource_tooltip_plugins.cpp

```diff
@@ -107,7 +107,8 @@ Control *EditorTextureTooltipPlugin::make_tooltip_for_path(const String &p_resou
 	vb->set_alignment(BoxContainer::ALIGNMENT_CENTER);
 
 	Vector2 dimensions = p_metadata.get("dimensions", Vector2());
-	Label *label = memnew(Label(vformat(TTR(U"Dimensions: %d × %d"), dimensions.x, dimensions.y)));
+	static constexpr char32_t dimensions_label_text[] = TTRC(U"Dimensions: %d × %d");
+	Label *label = memnew(Label(vformat(TTR(String(dimensions_label_text)), dimensions.x, dimensions.y)));
```

Comment:
```suggestion
	Label *label = memnew(Label(vformat(TTR(dimensions_label_text), dimensions.x, dimensions.y)));
```

Reply:
This is not possible, because `dimensions_label_text` is `char32_t *`. `StringName` does not currently have a `char32_t *` constructor.

Reply:
Ah OK my bad! Seems fine to me then ^^